pcap.c

```diff
@@ -278,7 +278,7 @@ pcap_PcapObject_read(PyObject *p)
       }
 
       if (len != sizeof(*hdr)) {
-        PyErr_Format(PyExc_IOError, "Read returned wrong number of bytes (%d)", len);
+        PyErr_Format(PyExc_IOError, "Read returned wrong number of bytes (%lu)", len);
```

Comment:
According to the Python documentation, it looks like this should actually be "%zd".